src/logic/playersmanager.h

```diff
@@ -92,12 +92,15 @@ class PlayersManager {
 	void add_player_end_status(const PlayerEndStatus& status, bool change_existing = false);
 
 	[[nodiscard]] const PlayerEndStatus* get_player_end_status(PlayerNumber player) const;
+	[[nodiscard]] const std::map<PlayerNumber, PlayerEndStatus>& get_players_end_status() {
```

Comment:
may we name this function get_all_players_end_status to avoid logical error due to a silly typo. (or any other name more distinguishable)

Reply:
Indeed. Thanks.